### DISCUSSION TITLE
Welcome to ScoopBox Discussions!

### DISCUSSION THREAD

**Discussion by hasan-hasanov** [hasan-hasanov/ScoopBox#4] [General]
## 👋 Welcome and thanks for your interest in this project!
This is a place to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Don't be shy. Ask something.

  To get started, comment below with an introduction of yourself and preferably write a joke too.

**Comment by DRFTA** [comment]
Hi mate, i made a script for sandbox that auto adds programs to auto install every time at start up. You might be able to use some of it in your repo. keep up the good work. https://github.com/DRFTA/Sandbox-auto-app-installer
